src/compiler/csharp_generator_helpers.h

```diff
@@ -22,13 +22,30 @@
 #include "src/compiler/config.h"
 #include "src/compiler/generator_helpers.h"
 
+using google::protobuf::compiler::csharp::GetOutputFile;
```

Comment:
@tonydnewell we needed to revert this in https://github.com/grpc/grpc/pull/32636 because of some internal build errors, basically the following:

```
src/compiler/csharp_generator_helpers.h:25] error: no member named 'compiler' in namespace 'google::protobuf'
using google::protobuf::compiler::csharp::GetOutputFile;
```

I have not gotten too far into the details but I'm wondering if the `google::protobuf::compiler::csharp::GetOutputFile` is a new API, only in some older versions of protobuf, or something else like that.